arangod/IResearch/IResearchDataStore.cpp

```diff
@@ -2136,6 +1930,22 @@ void IResearchDataStore::initClusterMetrics() const {
   metric.add("arangodb_search_consolidation_time", gaugeToCoordinator);
 }
 
+void IResearchDataStore::createFlushSubscription() {
+  TRI_ASSERT(_flushSubscription == nullptr);
```

Comment:
```suggestion
    if (_flushSubscription == nullptr) {
```

Reply:
No, it's assert, it always no exist here